draft-ietf-quic-transport.md

```diff
@@ -4635,8 +4640,13 @@ preferred_address (0x000d):
   transport parameter is only sent by a server. Servers MAY choose to only send
   a preferred address of one address family by sending an all-zero address and
   port (0.0.0.0:0 or ::.0) for the other family. IP addresses are encoded in
-  network byte order. The CID Length field contains the length of the
-  Connection ID field.
+  network byte order.
+
+  The Connection ID field and the Stateless Reset Token field contain an
```

Comment:
This might fix the formatting issue.

```suggestion
: The Connection ID field and the Stateless Reset Token field contain an
```

Reply:
Thank you for the suggestion. Done in 7f34774.

Reply:
Thank you for the suggestion. Done in 7f34ff4.